berkeley-function-call-leaderboard/bfcl/eval_checker/executable_python_function.py

```diff
@@ -1,17 +1,17 @@
-import json
+import os
 import math
 import requests
 from custom_exception import NoAPIKeyError
 import time
 
+# Make sure the env variables are populated
+ENV_VARS = ("GEOCODE_API_KEY", "RAPID_API_KEY", "OMDB_API_KEY", "EXCHANGERATE_API_KEY")
```

Comment:
nit, ideally goes to a utils file or constants file

Reply:
yep. future PRs for constant file.